### ISSUE TITLE
RAML Editor Tools - Details 'required' does not get updated

### ISSUE THREAD

**Issue by sichvoge** [mulesoft/api-workbench#36]
With the following RAML

```yaml
#%RAML 1.0
title: Mondo API

traits:
  pagination:
    description: |
      Endpoints which enumerate objects support time-based and curser-based pagination.
    displayName: Pagination
    queryParameters:
      limit:
        type: integer
        maximum: 100
```

I get this:

![image](https://cloud.githubusercontent.com/assets/498114/10971489/6cf42556-83cc-11e5-8cfc-148bb0a0561d.png)

As you can see above, the required checkbox is not enabled. If I add `required: true` to the `limit` property. The box gets automatically enabled. Expected behaviour, but changing `true` to `false` does not uncheck the box anymore

![image](https://cloud.githubusercontent.com/assets/498114/10971586/f9e3df92-83cc-11e5-838e-6f9cc8b2332a.png)

I only get what I want when I remove the `required` property again.

**Comment by sichvoge** [comment]
Additionally to that, checking/unchecking one of these checkboxes does not update the RAML.

**Comment by ddenisenko** [comment]
Reproducible. Actually, unchecking the checkbox does remove "required: true" from the code, but does not add it back.

**Comment by petrochenko-pavel-a** [comment]
Fix will be published in 0.8.7
